setup.py

```diff
@@ -2,68 +2,82 @@
 
 from setuptools import find_packages, setup
 
-version = "1.36.0"
+if version_info[0] >= 3:
 
-with open('README.md') as file:
-    long_description = file.read()
+    setup(packages=find_packages(exclude=['ez_setup', 'examples', 'test', 'django_tests', 'django_tests.*']),
```

Comment:
It looks like this part can be moved to `pyproject.toml` as follows:
```
[tool.setuptools]
include-package-data = true
zip-safe = false

[tool.setuptools.packages.find]
exclude = ["samples", "tools", "test*", "django_tests*", "venv*"]
namespaces = false
```

Reply:
First of all: Thank you very much for the review! I also had migrated to this syntax before, but it gave warnings that the `tool.setuptools` section is still in beta, [see here](https://github.com/cloudinary/pycloudinary/pull/367/commits/79de139d66218806b3c9432049363ead4c0d42fa).
If you want, I can migrate them back, of course!

Reply:
@ThexXTURBOXx , It looks like BETA warning was only for Python3.7, Python 3.8 and up take a newer version of setuptools,  which works ok with that change.
Yes, you can safely migrate it back.

Reply:
Perfect, will do so tomorrow! Just came back home. Will ping when done, thanks for the feedback!